rotkehlchen/tests/api/test_users.py

```diff
@@ -354,6 +354,68 @@ def test_user_creation_with_already_loggedin_user(rotkehlchen_api_server, userna
     )
 
 
+# @pytest.mark.parametrize('start_with_logged_in_user', [True]) (NEEDED?)
```

Comment:
you can parametrize any fixture used. The `start_with_logged_in_user` is used and is True by default. You can see it here: https://github.com/rotki/rotki/blob/c0aae515e9d0a5e266ee4e35403b961ae175ca02/rotkehlchen/tests/fixtures/rotkehlchen.py#L23-L25